src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/entities/AsylumCaseDefinition.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities;
```

Comment:
Can't see a AsylumCaseDefinitionTest class.  Has this been missed?

Reply:
done